eng/pipelines/libraries/run-test-job.yml

```diff
@@ -55,11 +55,11 @@ jobs:
         testDisplayName: ${{ parameters.runtimeFlavor }}_interpreter_${{ parameters.liveRuntimeBuildConfig }}
 
       # To run the tests we just send to helix and wait, use ubuntu hosted pools for faster providing and to not back up our build pools
-      ${{ if startsWith(parameters.pool.queue, 'BuildPool.Ubuntu') }}:
+      ${{ if startsWith(parameters.pool.queue, 'Build.Ubuntu') }}:
```

Comment:
I'm not sure I agree with this sentiment.  I favor as many machines as can be of the 200 parallelism limit be macOS since ostensibly 1ES can handle these but not MacOS

Reply:
This change is not correct anyway, as the new `pool` property is `demands` and it doesn't start with `Build.Ubuntu`. 

`pool.queue` no longer exists after this change. So we can safely delete this as now are a no-op.